docs-web/Makefile

```diff
@@ -0,0 +1,28 @@
+NETLIFY_ROOT	=	/opt/build/repo
+SITE_ROOT			= ${NETLIFY_ROOT}/site
+BUILDDIR			= ${NETLIFY_ROOT}/docs-web/build
+#DOCKER_IMAGE 	=
```

Comment:
the Makefile includes a few commented out parts. Are those necessary to keep or could be removed?